stackscli.yml

```diff
@@ -0,0 +1,220 @@
+      desc: Creating the archetype resources using the cqrs+events source code
+
+    - action: cmd
+      cmd: mvn
+      args: install
+              -f ./java/target/generated-sources/archetype/pom.xml
+              --quiet
+      desc: Installing the archetype
+              (groupId= com.amido.stacks.workloads, artifactId= stacks-api-cqrs-events-archetype, version= 1.0.0) in the local maven repository
+
+    - action: cmd
+      cmd: mvn
+      args: archetype:generate
+              -DoutputDirectory={{ .Project.Directory.WorkingDir }}
+              -DarchetypeGroupId=com.amido.stacks.workloads
+              -DarchetypeArtifactId=stacks-api-cqrs-events-archetype
+              -DarchetypeVersion=1.0.0
+              -DgroupId=com.{{ .Input.Business.Company}}.{{ .Input.Business.Domain }}
+              -DartifactId={{ .Project.Name }}
+              -Dpackage=com.{{ .Input.Business.Company}}.{{ .Input.Business.Domain }}.{{.Input.Business.Component}}
+              --quiet
+              -B
+      desc: Generating the cqrs+events project
+
+    - action: cmd
+      cmd: mvn
+      args: process-resources
+              -f stacksclipom.xml
+              -P rename-source-folder
+              -Dtemp.project.dir={{ .Project.Name }}
+              -Dtarget.dir={{ .Project.Directory.WorkingDir }}
+              --quiet
+      desc: Renaming the source code folder to java
+
+    - action: cmd
+      cmd: mvn
+      args: process-resources
+              -f {{ .Project.Directory.WorkingDir }}/java/pom.xml
+              -P update-permissions
+              --quiet
+      desc: Updating file permissions
+
+    - action: cmd
+      cmd: mvn
+      args: fmt:format
+              -f {{ .Project.Directory.WorkingDir }}/java/pom.xml
+              --quiet
+      desc: Formating the cqrs+events java code to meet the fmt standards
+
+    - action: cmd
+      cmd: mvn
+      args: dependency:resolve
+              -f {{ .Project.Directory.WorkingDir }}/java/pom.xml
+              --quiet
+      desc: Downloading maven dependencies for the cqrs+events project
+
+    # Build archetype for the serenity tests
+    - action: cmd
+      cmd: mvn
+      args: process-resources
+              -f ./api-tests/pom.xml
+              -P archetype-resources
+              --quiet
+      desc: Duplicating the original pom.xml file to archetype-pom.xml
+
+    - action: cmd
+      cmd: mvn
+      args: clean archetype:create-from-project
+              -f ./api-tests/pom-temp.xml
+              -DpropertyFile=./api-tests/archetype.properties
+              --quiet
+      desc: Creating the archetype resources using the api-test source code
+
+    - action: cmd
+      cmd: mvn
+      args: install
+              -f ./api-tests/target/generated-sources/archetype/pom.xml
+              --quiet
+      desc: Installing the archetype (groupId= com.amido.stacks.tests, artifactId= stacks-api-cqrs-events-tests-archetype, version= 1.0.0) in the local maven repository
```

Comment:
same as above

Reply:
Test references are already included in the artifactId and the groupId, I would keep it as it is